### PR TITLE
fix(demos): change demodata signed urls to serve minio bytes instead

### PR DESCRIPTION
**Pull request by jayceslesar**
We were polluting the domain namespace by putting making a minio bucket of `demos`. The API wanted us to hit domain/demos/some_url... but that overlaps with domain/demos/... which is the websocket domain.

This change was reverted in favor of just serving the minio objects as File Responses in litestar and is deemed okay because the strain the server was feeling before was due to postgres having to paginate the large object table whereas now we are just serving bytes back from minio.

### CODE REVIEW

File: masterbase/app.py [hunk 1]
```diff
@@ -156,15 +156,19 @@ def list_demos(
...
+    blob_name = demo_blob_name(session_id)
+    file = minio_client.get_object("demoblobs", blob_name)
+    stat = minio_client.stat_object("demoblobs", blob_name)
+
+    headers = {
+        "Content-Disposition": f'attachment; filename="{blob_name}"',
+        "Content-Length": str(stat.size),
+    }
+
+    return Stream(file.stream(), media_type=MediaType.TEXT, headers=headers)
```
Comment (kavorite) [inline masterbase/app.py:171]: elegant 👍

File: vars.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export DEVELOPMENT=true
```
Comment (kavorite) [inline vars.sh:1]: bout time someone committed this crap

**Review by kavorite — APPROVED** [review]
LGTM 🚀

File: vars.sh (hunk 2 above)
Comment (jayceslesar) [reply vars.sh:1]: hell yeah

**Pull request merged**